stdlib/database/sql/src/main/java/org/ballerinalang/database/table/BMirrorTable.java

```diff
@@ -86,8 +86,8 @@ public void addData(BMap<String, BValue> data, Context context) {
             PreparedStatement stmt = conn.prepareStatement(sqlStmt);
             TableUtils.prepareAndExecuteStatement(stmt, data);
             reset(isInTransaction);
-        } catch (SQLException e) {
-            throw new BallerinaException("execute add failed: " + e.getMessage(), e);
+        } catch (Throwable e) {
```

Comment:
Why are we catching throwable here?

Reply:
There can be situations where other exceptions are thrown. For ex: https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-core/src/main/java/org/ballerinalang/util/TableUtils.java#L131